inc/Engine/CDN/RocketCDN/ServiceProvider.php

```diff
@@ -38,25 +38,25 @@ class RocketCDN extends AbstractServiceProvider {
 	 */
 	public function register() {
 		// RocketCDN API Client.
-		$this->getContainer()->add( 'rocketcdn_api_client', 'WP_Rocket\CDN\RocketCDN\APIClient' );
+		$this->getContainer()->add( 'rocketcdn_api_client', 'WP_Rocket\Engine\CDN\RocketCDN\APIClient' );
 		// RocketCDN CDN options manager.
-		$this->getContainer()->add( 'rocketcdn_options_manager', 'WP_Rocket\CDN\RocketCDN\CDNOptionsManager' )
+		$this->getContainer()->add( 'rocketcdn_options_manager', 'WP_Rocket\Engine\CDN\RocketCDN\CDNOptionsManager' )
 			->withArgument( $this->getContainer()->get( 'options_api' ) )
 			->withArgument( $this->getContainer()->get( 'options' ) );
 		// RocketCDN Data manager subscriber.
-		$this->getContainer()->share( 'rocketcdn_data_manager_subscriber', 'WP_Rocket\Subscriber\CDN\RocketCDN\DataManagerSubscriber' )
+		$this->getContainer()->share( 'rocketcdn_data_manager_subscriber', 'WP_Rocket\Engine\CDN\RocketCDN\DataManagerSubscriber' )
 			->withArgument( $this->getContainer()->get( 'rocketcdn_api_client' ) )
 			->withArgument( $this->getContainer()->get( 'rocketcdn_options_manager' ) );
 		// RocketCDN REST API Subscriber.
-		$this->getContainer()->share( 'rocketcdn_rest_subscriber', 'WP_Rocket\Subscriber\CDN\RocketCDN\RESTSubscriber' )
+		$this->getContainer()->share( 'rocketcdn_rest_subscriber', 'WP_Rocket\Engine\CDN\RocketCDN\RESTSubscriber' )
 			->withArgument( $this->getContainer()->get( 'rocketcdn_options_manager' ) )
 			->withArgument( $this->getContainer()->get( 'options' ) );
 		// RocketCDN Notices Subscriber.
-		$this->getContainer()->share( 'rocketcdn_notices_subscriber', 'WP_Rocket\Subscriber\CDN\RocketCDN\NoticesSubscriber' )
+		$this->getContainer()->share( 'rocketcdn_notices_subscriber', 'WP_Rocket\Engine\CDN\RocketCDN\NoticesSubscriber' )
 			->withArgument( $this->getContainer()->get( 'rocketcdn_api_client' ) )
 			->withArgument( $this->getContainer()->get( 'template_path' ) . '/settings/rocketcdn' );
 		// RocketCDN settings page subscriber.
-		$this->getContainer()->share( 'rocketcdn_admin_subscriber', 'WP_Rocket\Subscriber\CDN\RocketCDN\AdminPageSubscriber' )
+		$this->getContainer()->share( 'rocketcdn_admin_subscriber', 'WP_Rocket\Engine\CDN\RocketCDN\AdminPageSubscriber' )
 			->withArgument( $this->getContainer()->get( 'rocketcdn_api_client' ) )
 			->withArgument( $this->getContainer()->get( 'options' ) )
 			->withArgument( $this->getContainer()->get( 'beacon' ) )
```

Comment:
Let's assign the `options` handler to a variable and then inject the variable.

Why?

`$this->getContainer()->get()` creates a new instance when invoked. Options then has to go grab the options from cache or the db to populate. It's more peformant to do that once for a given module.